build.gradle

```diff
@@ -147,7 +147,9 @@ project ('spring-kafka') {
 
 	dependencies {
 		compile "org.springframework:spring-messaging:$springVersion"
-		compile "org.apache.kafka:kafka-clients:$kafkaVersion"
+		compile ("org.apache.kafka:kafka-clients:$kafkaVersion") {
+			exclude group: 'org.slf4j'
```

Comment:
Well, exclude just `group` isn't enough for other our requirements like [Spring IO Platform](http://platform.spring.io/platform/). But that is already not your headache.
I'm merging it as is and think about dependencies offline.
Seems form me we should retain `slf4j` as `runtime` to let Kafka to log in our tests anyway.